lib/subtitles.ex

```diff
@@ -1,3 +1,5 @@
 defmodule Subtitles do
-
+  def get_format(subtitle) do
+    subtitle |> String.starts_with?("WEBVTT") && :vtt || :srt
```

Comment:
You should default to unknown.

For example vtt will always start with WEBVTT true but srt will always start with a 1 and the CRLF. 

It should default to unknown.

Also you should give room for adding new formats easily.

```
subtitle |> is_srt? && :srt ||
subtitle |> is_vtt? && :vtt ||
:unknown
```

Reply:
Good points!
CRLF is however not required for SRT, any newline is valid from what I can find.